### PR TITLE
fix resource to only quote values with spaces

### PR DESCRIPTION
**Pull request by rmcleod8**

### CODE REVIEW

File: resources/config.rb [hunk 1]
```diff
@@ -16,7 +16,7 @@ def content
     next n.flatten.map(&:to_s).join(',') if n.is_a?(Array)
     n
   end.map do |k, v|
-    'GRUB_' << k.upcase << (v.is_a?(String) ? "=\"#{v}\"" : "=#{v}")
+    'GRUB_' << k.upcase << (v =~ /[\W\s]+/ ? "=\"#{v}\"" : "=#{v}")
```
Comment (jjustice6) [inline resources/config.rb]: Should this clause be `v.is_a?(String) && v =~ /[\W\s]+/` ?

Comment (jjustice6) [reply resources/config.rb]: Note:
```
irb(main):005:0> v = 'has a space'.to_sym
=> :"has a space"
irb(main):006:0> v =~ /[\W\s]+/
=> 3
```

Comment (rmcleod8) [reply resources/config.rb]: thanks @jjustice6 updated

Comment (jjustice6) [reply resources/config.rb]: This LGTM now

**Pull request merged**